docs/django.md

```diff
@@ -113,6 +113,13 @@ In this step, you create a Django started project by building the image from the
         $ ls
```

Comment:
This is a note that should be part of  a step. Start by correcting step 3 to do an `ls -al`  Then...under that put:

  The files `django-admin` created are owned by root. This happens because the
  container runs as the `root` user.  

 4. Change the ownership of the new files.

      `sudo chown -R $USER:$USER .`